helpers/seed.go

```diff
@@ -0,0 +1,114 @@
+package helpers
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/gofrs/uuid"
+	models "github.com/layer5io/meshery/models"
+	"github.com/sirupsen/logrus"
+)
+
+type contentandName struct {
+	name    string
+	content string
+}
+type Component string
+
+const (
+	Filter  Component = "FILTER"
+	Pattern Component = "PATTERN"
+)
+
+var seededUUIDs []uuid.UUID
+var localProvider *models.DefaultLocalProvider
```

Comment:
Any reason for not using the original reference to `DefaultLocalProvider` and creating this global variable? I think we can pass on that reference to other functions too and avoid this new var. :thinking:

Reply:
it was a mistake